pkg/csi/service/wcp/controller.go

```diff
@@ -844,18 +844,26 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 func (c *controller) createFileVolume(ctx context.Context, req *csi.CreateVolumeRequest) (
```

Comment:
You will also need to start running CNSFileAccessConfig controller for RWX provisioning to work. That is currently disabled for multi-zone SV.

Reply:
done, updated code to enable CNSFileAccessConfig controller and also install required CRDs